code/modules/uplink/uplink_items.dm

```diff
@@ -219,7 +219,6 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			agent card, specialised contractor baton, and three randomly selected low cost items. Can include otherwise unobtainable items."
 	item = /obj/item/storage/box/syndicate/contract_kit
 	cost = 20
-	player_minimum = 20
```

Comment:
I think contractor should still have a player minimum, it doesn't really work at higher pop